pkg/resourceskeleton/testdata/uri-skeleton.yaml

```diff
@@ -791,6 +791,17 @@
         external: bigquerydatasetsamplehnf4t2squ1g1ghnendt0
   ResourceConfigId: BigQueryTable
   URI: "https://bigquery.googleapis.com/bigquery/v2/projects/kcc-test/datasets/bigquerydatasetsamplehnf4t2squ1g1ghnendt0/tables/bigquerytablesamplehnf4t2squ1g1ghnendt0"
+- ExpectedSkeleton:
+    apiVersion: bigqueryconnection.cnrm.cloud.google.com/v1beta1
```

Comment:
`v1alpha1`. This is not a v1beta1 resource.

Reply:
thanks!